apps/music/test/marionette/Player_test.js

```diff
@@ -131,6 +136,36 @@ marionette('Music player tests', function() {
       music.tapHeaderActionButton();
       music.checkPlayerIconShown(true);
     });
+
+    test('Check the play icon is in the status bar', function() {
+      music.launch();
+      music.waitForFirstTile();
+      music.switchToSongsView();
+
+      // check the status bar for the hidden play icon
+      client.switchToFrame();
+      client.findElement('#statusbar-maximized');
```

Comment:
I don't understand what the findElement is doing here and below. If you really need it, maybe add a comment explaining why you're finding this element and discarding the result. Same comment below.